LICENSE.txt

```diff
@@ -11,7 +11,7 @@ Redistributions in binary form must reproduce the above copyright notice,
 this list of conditions and the following disclaimer in the documentation
 and/or other materials provided with the distribution.
 
-Neither the name of Continuum Analytics nor the names of any contributors
+Neither the name of Anaconda, Inc nor the names of any contributors
```

Comment:
@csoja, I just wanted to double check if this part of the license needs to be changed or it should be left as `Continuum Analytics`, do you know?

Reply:
Don't think we need to change all `Continuum Analytics` entries at this time... @csoja, can you confirm?

Reply:
It would be good to update them to the latest - for a license though - it should look similar to what we did in the Anaconda EULA: https://docs.anaconda.com/anaconda/eula
copyright changes to Anaconda, Inc.  Then, Continuum Analytics changed to  Continuum Analytics, Inc. (dba Anaconda, Inc.).  I would get Jeanie to review it before submitting the change though.  We want to update these as we come across them - but we won't hold up a release if this has not been updated.  (As legally the company is still Continuum Analytics)

Reply:
I think the eula example is what we should follow for license as well. Seems the most comprehensive. I will wait on the check :-)